rbx_tree/src/value.rs

```diff
@@ -18,4 +18,28 @@ pub enum RbxValue {
     Vector3 {
         value: [f64; 3],
     },
+    #[serde(rename_all = "PascalCase")]
+    Vector2 {
+        value: [f64; 2],
+    },
+    #[serde(rename_all = "PascalCase")]
+    Color3 {
+        value: [f32; 3],
+    },
+    #[serde(rename_all = "PascalCase")]
+    Color3uint8 {
+        value: [u8; 3],
+    },
+    #[serde(rename_all = "PascalCase")]
+    Vector3int16 {
+        value: [i16; 3],
+    },
+    #[serde(rename_all = "PascalCase")]
+    Vector2int16 {
+        value: [i16; 2],
+    },
+    #[serde(rename_all = "PascalCase")]
+    CoordinateFrame {
+        value: [f32; 12],
```

Comment:
I wonder if we should break this into position + rotation so that it's easier to reason about.